app/models/user.rb

```diff
@@ -33,4 +36,20 @@ def name_or_email
   def last_logged_in
     passwordless_sessions.try(:last).try(:claimed_at)
   end
+
+  def role_type
+    role&.role
```

Comment:
Not happy with role&.role.  Not sure what to name it exactly: the idea is that a Role has a friendly name, and than an internal identifier we can leave unchanged

Reply:
Normally I'd suggest using a non-integer primary key, but it looks like [rails doesn't like that](https://stackoverflow.com/questions/1200568/using-rails-how-can-i-set-my-primary-key-to-not-be-an-integer-typed-column)